projects/angular/src/timeline/_variables.timeline.scss

```diff
@@ -3,63 +3,23 @@
  * This software is released under MIT license.
  * The full license information can be found in LICENSE in the root directory of this project.
  */
-@use '../utils/variables/variables';
+@use '@cds/core/tokens/tokens.scss';
 
-$clr-timeline-line-color: variables.$clr-color-neutral-500 !default; // #9a9a9a
-$clr-timeline-step-header-color: variables.$clr-color-neutral-600 !default; // #737373
-$clr-timeline-step-title-color: variables.$clr-color-neutral-700 !default; // #565656
-$clr-timeline-step-description-color: variables.$clr-color-neutral-700 !default; // #565656
+$clr-timeline-line-color: var(--clr-timeline-line-color) !default;
+$clr-timeline-step-header-color: var(--clr-timeline-step-header-color) !default;
+$clr-timeline-step-title-color: var(--clr-timeline-step-title-color) !default;
+$clr-timeline-step-description-color: var(--clr-timeline-step-description-color) !default;
 
-$clr-timeline-incomplete-step-color: variables.$clr-color-neutral-600 !default;
-$clr-timeline-current-step-color: variables.$clr-color-action-600 !default;
-$clr-timeline-success-step-color: variables.$clr-color-success-400 !default;
-$clr-timeline-error-step-color: variables.$clr-color-danger-800 !default;
-
-$clr-timeline-incomplete-step-color: variables.$clr-color-neutral-600 !default;
-$clr-timeline-current-step-color: variables.$clr-color-action-600 !default;
-$clr-timeline-success-step-color: variables.$clr-color-success-400 !default;
-$clr-timeline-error-step-color: variables.$clr-color-danger-800 !default;
-
-// 1rem = 24px
+$clr-timeline-incomplete-step-color: var(--clr-timeline-incomplete-step-color) !default;
+$clr-timeline-current-step-color: var(--clr-timeline-current-step-color) !default;
+$clr-timeline-success-step-color: var(--clr-timeline-success-step-color) !default;
+$clr-timeline-error-step-color: var(--clr-timeline-error-step-color) !default;
 
 // overall
-$clr-timeline-icon-size: variables.$clr_baselineRem_1_5;
-$clr-timeline-spinner-size: variables.$clr_baselineRem_1_33;
-$clr-timeline-default-icon-size: variables.$clr_baselineRem_0_667;
-$clr-timeline-icon-inner-padding: variables.$clr_baselineRem_2px;
-$clr-timeline-padding-width: variables.$clr_baselineRem_0_5;
-$clr-timeline-line-thickness: variables.$clr_baselineRem_2px;
-$clr-timeline-gutter-width: variables.$clr_baselineRem_0_5;
-
-// timeline step (horizontal)
-$clr-timeline-step-min-width: variables.$clr_baselineRem_7_29167;
-
-// header (timestamp)
-$clr-timeline-step-header-font-size: variables.$clr-p2-font-size;
-$clr-timeline-step-header-line-height: variables.$clr_baselineRem_0_75;
-$clr-timeline-step-header-padding-bottom: variables.$clr_baselineRem_8px;
+$clr-timeline-icon-size: #{tokens.$cds-global-space-11};
```

Comment:
I am not sure why we have these as variables in the variables file - maybe use the cds tokens inside the timeline.clarity file?

Reply:
`$clr-timeline-icon-size`, `$clr-timeline-line-thickness` and `$clr-timeline-default-icon-size` are variables for internal use only and to see how some rues are calculated.